components/test/test_subprocess.py

```diff
@@ -85,8 +85,14 @@ def f(
             return y
 
         _, body = task_base.parse_f(f)
+        # Python 3.7 removes docstring but 3.8+ doesn't. See `parse_f` for details.
+        docstring = """\
```

Comment:
Nit: You can avoid some escaping by using single quotes. (e.g. `''' """Stuff""" '''`)